rcsb/exdb/chemref/PubChemEtlWrapper.py

```diff
@@ -117,7 +117,8 @@ def updateIndex(self, **kwargs):
             rebuildChemIndices (bool, optional): rebuild indices from source (default: False)
             fetchLimit (int, optional): maximum number of definitions to process (default: None)
             exportPath(str, optional): path to export raw PubChem search results  (default: None)
-            numProc(int):  number processors to include in multiprocessing mode (default: 12)
+            numProcChem(int):  number processors to include in multiprocessing mode (default: 12)
+            numProc(int):  number processors to include in multiprocessing mode (default: 1)
```

Comment:
So just to clarify, is the one that we need to be cautious with the `numProcChem`? Or `numProc`?

If it's `numProcChem`, then we should probably lower the default down to whatever is the safest value (i.e., one which won't cause "too many request" errors). Also, if `numProc` can be increased, maybe increase the default to 2? In either case, I don't think we should ever set the default procs to 12, since most of our instances only have 8 at most.

If you can differentiate these parameters a little more clearer, that would be helpful. (BTW there's a space missing between the parameter name and data type).